client-app/src/pages/account/store/modules/orders-list/mutations.ts

```diff
@@ -17,5 +21,15 @@ export const mutations: MutationTree<OrdersListState> = {
   },
   [SET_ORDERS_LIST_CONFIG](state, payload: OrdersListConfig) {
     state.ordersList.listConfig = payload;
+  },
+  [SET_ORDER](state, payload: CustomerOrder) {
+    state.selectedOrder = payload;
+    state.loaded = true;
+    state.isLoading = false;
+    state.selectedOrderIsLoaded = true;
+  },
+  [CLEAR_ORDER_DETAIL](state) {
```

Comment:
Rename to CLEAR_SELECDTED_ORDER. Action name does not match the state`s property name "selectedOrder".